RELEASE_NOTES.md

```diff
@@ -1,5 +1,28 @@
 # Riak Python Client Release Notes
 
+## 2.3.0 Feature Release - 2015-12-14
+
+Release 2.3.0 features support for new
+[time series](https://github.com/basho/riak-python-client/pull/416)
+functionality.
+
+This is release retires support for Python 2.6.x but adds support for
+Python 3.5.x.
+
+There are also many bugfixes and new enhancements:
+
+* [Protocol buffers are now integrated into the Python Client]
+  (https://github.com/basho/riak-python-client/pull/418)
+* [Support for Preflists and Write-Once bucket types]
+  (https://github.com/basho/riak-python-client/pull/414)
+* [Support Riak 2.1.1]
+  (https://github.com/basho/riak-python-client/pull/407)
+* [Addition of new API entry point methods]
+  (https://github.com/basho/riak-python-client/pull/398)
```

Comment:
https://github.com/basho/riak-python-client/pull/398 Was never actually merged.... did it make it in some other way?